### ISSUE TITLE
Questions and feedback

### ISSUE THREAD

**Issue by geeewizzz** [karloygard/hassio-addons#3]
I'm a bit of a noob when it comes to HA though, but let me try.

1. Seems like all my heating actuaturs come up as "light.xcomfort_deviceid..." , probably should be named differently.
2. It would be nice to easily add "climate" devices in home assistant, but I assume that it's difficult to match up automatically when installing the add-on because there is no way to know which sensor belongs to which heating actuator?
3. I tried adding one manually in the configuration:

```
 climate:
   - platform: generic_thermostat
     name: Liten gang
     heater: light.xcomfort_5733608_1_etg_liten_gang_vk
     target_sensor: sensor.xcomfort_5881353_1_etg_liten_gang_temp_kanal_b
```
Seems to be working, other than the fact if I turn up the wanted temp to a temp above the actual temp right now it doesn't start heating (which I hoped it would). 

<img width="474" alt="Screenshot 2020-04-13 at 20 23 48" src="https://user-images.githubusercontent.com/4536023/79147887-f7a3f180-7dc4-11ea-9de3-c5a247a1c780.png">

**Comment by karloygard** [comment]
1. The naming is pretty random at the moment, suggestions appreciated. Or did you want it to be `switch` instead of `light`?

2. I *think* it's technically possible. If the datapoints file is exported in the DPL format, some topology is included, which might make it possible to infer how things are connected. However, I have no documentation of that file format, and I haven't attempted to reverse engineer it, so for the time being, you have to connect them manually.

3. Can you see in the logs if it's attempting to send messages to the actuators? It certainly seems like it should work, but I haven't fiddled with `generic_thermostat`, so I don't know if it's sending something strange. Does turning actuators on/off manually in the Home Assistant UI work?

**Comment by geeewizzz** [comment]
1. Yeah, switch would probably make a lot more sense i guess?
2. Thats ok, thanks.
3. 
`2020/04/14 07:11:01 topic: xcomfort/22/set/switch, message: true
2020/04/14 07:11:02 rx: [c11670000100000000461004]
2020/04/14 07:11:02 Device 5733608 (channel 0-'1.etg liten gang vk') sent message (battery mains-powered, signal normal, cyclic false) status switched on
2020/04/14 07:11:30 topic: xcomfort/22/set/switch, message: false
2020/04/14 07:11:31 rx: [c11670000000000000431006]
2020/04/14 07:11:31 Device 5733608 (channel 0-'1.etg liten gang vk') sent message (battery mains-powered, signal good, cyclic false) status switched off`

So manually turning it on off in "generic thermostat" works fine, but turning up the wanted temperature does nothing it seems.  I'll play around with other thermostats and see if they work differently.

**Comment by karloygard** [comment]
Could be that generic thermostat would play nicer with a switch instead of a light. I'll test and see later today.

**Comment by karloygard** [comment]
By the way, did you turn the thermostat on (click the little flame)?

**Comment by geeewizzz** [comment]
Ah damn, now i feel stupid. Thought that was "heater on/off". It works. I thought that flame would come on when i turned the desired temp over the current sensor temp.

Can I claim confusing UX? :)

**Comment by karloygard** [comment]
No worries, took me ten minutes to figure that out too. 🥇 Agreed, HA has a ways to go wrt. usability still.

**Comment by geeewizzz** [comment]
Possibly another dumb question, when restarting my home assistant all my devices and entities from this addon become unavailable, it seems i need to restart the addon manually for them to come back. Is this something I can configure somewhere to avoid?

**Comment by karloygard** [comment]
I see the same issues here; could be the MQTT discovery that's somehow running into racing issues. I'll investigate when I have time.

**Comment by geeewizzz** [comment]
A work-around:
<img width="981" alt="Screenshot 2020-04-16 at 12 53 04" src="https://user-images.githubusercontent.com/4536023/79448214-52199980-7fe1-11ea-8385-c2ecc5ca45f5.png">

**Comment by geeewizzz** [comment]
[log.txt](https://github.com/karloygard/hassio-addons/files/4502632/log.txt)
[usbstick_full6.txt](https://github.com/karloygard/hassio-addons/files/4502633/usbstick_full6.txt)

Seems like additional sensors from Room Controller touch is not being added as entities?
Added all sensors from 1.etg_dusj/bad now and seems like it is ignoring some of them.

The RCT has the following sensors :
Temperatur (which is the devices own temperature sensor, room temp)
Fuktighet (measures humidity in the room)
Ekstern sensor(in my case measures the floor temp)

In addition 2 buttons that you can also connect.

It seems MRF has limitations here, for the RCT sensors it can only connect to 1 controller. I didnt realize that until this weekend. So since mine were connected to Sensio from before, I removed the sensio connection for this one device and now connected the all sensors to the USB stick instead.

**Comment by geeewizzz** [comment]
Another bug in Eaton MRF: for some temperature sensors it seems to on random want to set them to send switching commands even though I specify in MRF that I want it to send temperature commands.

I have changed this manually in the data point file where I figured it is the first 0/1 that is controlling this. Since the switching command is not supported anyway, maybe it is wise to assume that to be a mistake and import it as a temperature sensor anyway? (and just print a warning in the logfile or something)
